packages/core/src/utilities/renderToCanvasGPU.ts

```diff
@@ -39,28 +39,33 @@ export default function renderToCanvasGPU(
   const viewportId = `renderGPUViewport-${imageIdToPrint}`;
   const imageId = image.imageId;
   const element = document.createElement('div');
-  element.style.width = `${canvas.width}px`;
-  element.style.height = `${canvas.height}px`;
+  const devicePixelRatio = window.devicePixelRatio || 1;
+  const originalWidth = canvas.width;
+  const originalHeight = canvas.height;
+  // The canvas width/height are set by flooring the CSS size converted
+  // into physical pixels, but because these are float values, the conversion
+  // isn't exact, and using the exact value sometimes leads to an off by 1
+  // in the actual size, so adding a half physical pixel to the size resolves
+  // the problem.
+  element.style.width = `${originalWidth + devicePixelRatio / 2}px`;
```

Comment:
why this is different than the one in `getOrCreateCanvas`, I know one is element and the other is canvas but this one is doign `devicePixelRatio/2` 

```js
canvas.style.width = (width + 0.0001) / devicePixelRatio + 'px';
canvas.style.height = (height + 0.0001) / devicePixelRatio + 'px';
```

Reply:
Should I change it to + 0.0001 as well?  Anything smaller than the device pixel ratio and larger than the next fractional size should be ok.

Reply:
why one of them has devided by two and other does not? Shouldn't it be consistent?